src/handlers/int-test-support/handler.ts

```diff
@@ -0,0 +1,70 @@
+import { Context } from "aws-lambda";
+import {
+  listS3Objects,
+  getS3Object,
+  deleteS3Objects,
+  deleteS3Object,
+  getS3Objects,
+  checkIfS3ObjectExists,
+  putS3Object,
+} from "./helpers/s3Helper";
+import { publishSNS } from "./helpers/snsHelper";
+import { checkGivenStringExistsInLogs } from "./helpers/cloudWatchHelper";
+import {
+  getQueryExecutionStatus,
+  startQueryExecutionCommand,
+  getQueryResults,
+} from "./helpers/athenaHelper";
+import { createInvoiceInS3 } from "./helpers/mock-data/invoice/helpers";
+
+export interface TestSupportEvent {
+  environment: string;
+  command: string;
+  parameters: any;
+}
+
+export interface TestSupportReturn {
+  error?: string;
+  success?: boolean;
+  successObject?: any;
+}
+
+const functionMap: { [name: string]: Function } = {
```

Comment:
probably fine in this case cus it's for test tooling but I'd normally discourage the use of `Function` as a type

Reply:
I agree with you fully, but this would be a huge mess of code without it.